src/resource/type/Light.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <cstdint>
+#include "resource/Resource.h"
+
+namespace LUS {
+
+struct LightN64 {
+    uint8_t Color[3];
+    int8_t Pad1;
+    uint8_t ColorCopy[3];
+    int8_t Pad2;
+    uint8_t Direction[3];
+    int8_t Pad3;
+};
```

Comment:
having the padding in the resource types doesn't really fit with existing patterns. i'd think instead the idea would be to move that complexity into the importer

Reply:
The thing with this is that we dont need the gbi, because this is exactly what we do on Gfx and Mtx, but we take the type from the gbi